src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -235,7 +253,11 @@ export const useSwapper = () => {
     if (!wallet) throw new Error('no wallet available')
 
     const tradeQuote = await (async () => {
-      if (sellAsset.chainId === KnownChainIds.EthereumMainnet) {
+      if (
+        sellAsset.chainId === KnownChainIds.EthereumMainnet ||
+        sellAsset.chainId === KnownChainIds.OsmosisMainnet ||
+        sellAsset.chainId === KnownChainIds.CosmosMainnet
+      ) {
```

Comment:
similarly, this should be like `if (isSupportedSwappingChain(sellAsset.chainId))`